sdks/python/apache_beam/io/fileio.py

```diff
@@ -554,6 +561,18 @@ def process(self, element):
                        r.pane,
                        destination)
 
+    logging.info('Cautiously removing temporary files for destination %s',
+                 destination)
+    self._remove_temporary_files([r.file_name for r in file_results])
```

Comment:
There's an implicit `return None` at the end of this method. Are `None`s handled correctly?

Reply:
I believe this method will return a generator due to the above yield keyword, so this does not return None.